code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm

```diff
@@ -89,4 +89,70 @@
 			victim.apply_effect(6, STUTTER, 0)
 			victim.take_organ_damage(10)
 
-		occupant.visible_message("<span class='danger'>[occupant] crashed into \the [A]!</span>")
\ No newline at end of file
+		occupant.visible_message("<span class='danger'>[occupant] crashed into \the [A]!</span>")
+
+/obj/structure/stool/bed/chair/wheelchair/bike
+	name = "bicycle"
+	desc = "Two wheels of FURY!"
+	//placeholder until i get a bike sprite
+	icon = 'icons/vehicles/motorcycle.dmi'
+	icon_state = "motorcycle_4dir"
+
+/obj/structure/stool/bed/chair/wheelchair/bike/relaymove(mob/user, direction)
+	if(propelled)
+		return 0
+
+	if(!Process_Spacemove(direction) || !has_gravity(src.loc) || !isturf(loc))	//bikes in space.
+		return 0
+
+	if(world.time < move_delay)
+		return
+
+	var/calculated_move_delay
+	calculated_move_delay = 0 //bikes are infact sport bikes
```

Comment:
probably, almost made it that but then would have to deal with keys and batteries and such.

wheelchair seemed easier (and i'm lazy to a fault at times)